security-analysis/security-analysis-afs/src/main/java/com/powsybl/security/afs/SecurityAnalysisRunner.java

```diff
@@ -130,6 +130,12 @@ public void writeResult(SecurityAnalysisResult result) {
         storage.flush();
     }
 
+    public void clearResults(){
```

Comment:
Should we keep the  s as we don't have  it in any other methods (and it is probably a mistake...)